docs/content/concepts/assets/asset-checks.mdx

```diff
@@ -32,13 +32,14 @@ With asset checks, you can also:
 
 - Block downstream execution based on check results
 - Use severity levels and metadata [to communicate actionable details about check results in the UI](/concepts/assets/asset-checks/define-execute-asset-checks#using-asset-check-results)
+- [Execute subsets of checks](/concepts/assets/asset-checks/subsetting-asset-checks) to mitigate performance issues, such as those caused by an expensive check
```

Comment:
I think we should remove this line- it's it's not really a feature worth advertising, instead it's something you end up wanting if you use them in multi assets or multi asset checks

Reply:
Sounds good to me.